sormas-api/src/main/java/de/symeda/sormas/api/symptoms/SymptomsHelper.java

```diff
@@ -226,13 +234,13 @@ public static void updateSymptoms(SymptomsDto sourceSymptoms, SymptomsDto target
 					if (result == SymptomState.YES) {
 						pd.getWriteMethod().invoke(targetSymptoms, result);
 					}
-				} else if (pd.getReadMethod().getReturnType() == Boolean.class) {
+				} else if (pd.getReadMethod().getReturnType() == YesNoUnknown.class) {
```

Comment:
You're breaking existing logic here. Please revert this change. All you have to do is to remove the special condition for SymptomsDto.SYMPTOMATIC.